src/components/documents/DocumentHead.tsx

```diff
@@ -70,22 +71,10 @@ export const DocumentHead = ({ document, family, handleViewOtherDocsClick, handl
         />
       </SubNav>
       <SiteWidth>
-        <div className="flex flex-col lg:flex-row">
-          <div className="flex-1 my-4">
+        <div className="flex flex-col justify-between lg:flex-row">
+          <div className="flex-1 my-4 max-w-[1100px]">
```

Comment:
We shouldn't be setting containers in this way, that was the whole point of the `sr/components/panels` component set.

Can you use the `SingleCol` component?